src/test/java/com/spotify/dns/DnsSrvResolversIT.java

```diff
@@ -113,6 +117,16 @@ public void shouldFailForBadHostNames() throws Exception {
     }
   }
 
+  @Test
+  public void shouldReturnResultsUsingSpecifiedServers() throws Exception {
+    final String server = new SimpleResolver().getAddress().getHostName();
+    final DnsSrvResolver resolver = DnsSrvResolvers
+        .newBuilder()
+        .servers(ImmutableList.of(server))
+        .build();
+    assertThat(resolver.resolve("_spotify-client._tcp.spotify.com").isEmpty(), is(false));
```

Comment:
This test doesn't actually test that the given resolver is used, does it?

Reply:
You're correct that it doesn't verify that specified server was actually used. But it's invoking the resolve method and testing that code path, making sure it returns successfully. I'm not sure how to test that the resolver actually used the specified server.